### ISSUE TITLE
TargetPlatformVersion is 8.0 for win81 Target 

### ISSUE THREAD

**Issue by inforithmics** [onovotny/MSBuildSdkExtras#28]
When targeting win81 MSBuild Extras uses 8.0 Windows API so I changed in the following in the Windows.targets file.

```  
	<PropertyGroup Condition="'$(TargetFramework)' == 'win81'">
		<TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
		<NugetTargetMoniker>Windows,Version=v8.1</NugetTargetMoniker>
		<DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);NETFX_CORE;WINDOWS_APP</DefineConstants>

		<LanguageTargets>$(MSBuildThisFileDirectory)languageTargets\Windows.targets</LanguageTargets>
		<!-- Need to override the built-in implicit defines for win81 or it'll be NETCORE4_5_1 -->
		<ImplicitFrameworkDefine Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">WINDOWS8_1</ImplicitFrameworkDefine>
	</PropertyGroup>
```

To (Added TargetPlatformVersion 8.1)

```  
	<PropertyGroup Condition="'$(TargetFramework)' == 'win81'">
		<TargetPlatformVersion>8.1</TargetPlatformVersion>
		<TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
		<NugetTargetMoniker>Windows,Version=v8.1</NugetTargetMoniker>
		<DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);NETFX_CORE;WINDOWS_APP</DefineConstants>

		<LanguageTargets>$(MSBuildThisFileDirectory)languageTargets\Windows.targets</LanguageTargets>
		<!-- Need to override the built-in implicit defines for win81 or it'll be NETCORE4_5_1 -->
		<ImplicitFrameworkDefine Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">WINDOWS8_1</ImplicitFrameworkDefine>
	</PropertyGroup>
```

As Work around I use currently following Entry in my project file 

`<TargetPlatformVersion Condition="'$(TargetFramework)' == 'win81'">8.1</TargetPlatformVersion>`

**Comment by inforithmics** [comment]
Minimal Solution to reproduce it compiles with 8.1 Windows API fails with 8.0 Windows API

[TestWin81Build.zip](https://github.com/onovotny/MSBuildSdkExtras/files/1632023/TestWin81Build.zip)

**Comment by Nirmal4G** [comment]
`TargetPlatformVersion` should be added by Windows 8.x SDK targets

When I did the rework, I found that TPV was in fact added by the Windows `LanguageTargets`, so I removed those lines! It was working well for me. This ain't supposed to happen. I'll check the issue and get back to you!

**Comment by Nirmal4G** [comment]
temporary fix in #29

**Comment by onovotny** [comment]
Fixed in the latest code, thanks!
